app.js

```diff
@@ -0,0 +1,81 @@
+var express = require('express');
```

Comment:
Please use existing index.js file,
Please don't use express generator to create project structure,